features/partnerhistorie.feature

```diff
@@ -4,137 +4,259 @@ Functionaliteit: Tonen van Partnerhistorie
+  | 999991553           | 5         | Karel     | 20091102              |                          |                      |                               |
+  | 999991553           | 5         | Björn     |                       | 20080706                 |                      |                               |
+  | 999991553           | 55        | Björn     | 19870714              |                          |                      |                               |
+  | 999991553           | 55        | Björn     | 19870714              |                          |                      |                               |
+  | 999995182           | 5         | Erik      |                       | 19730812                 |                      |                               |
+  | 999995182           | 55        | Erik      |                       | 19730812                 |                      |                               |
+  | 999995182           | 55        | Erik      | 19650812              |                          |                      |                               |
+  | 999995182           | 5         | Henk      | 19740808              |                          |                      |                               |
+  | 999992806           | 5         | Osama     |                       | 20011109                 |                      |                               |
+  | 999992806           | 55        | Osama     | 20000115              |                          |                      |                               |
+  | 999991139           | 5         | Ayaan     | 19360000              |                          |                      |                               |
+  | 999991418           | 5         | Annechien | 00000000              |                          |                      |                               |
+  | 999992409           | 5         | Jennifer  | 20140123              |                          | 20160403             | 050000                        |
 
-  Wanneer de datum sluiting/aangaan gedeeltelijk onbekend is, wordt voor de filtering aangenomen dat de persoon gedurende de gehele onzekerheidstijd deze partner heeft gehad.
-  Wanneer van de sluiting/aangaan alleen het jaar bekend is, wordt voor de filtering aangenomen dat de persoon het hele jaar deze partner heeft gehad.
-  Wanneer van de sluiting/aangaan alleen het jaar en de maand bekend is, wordt voor de filtering aangenomen dat de persoon de hele maand deze partner heeft gehad.
-  Wanneer de sluiting/aangaan geheel onbekend is, wordt voor de filtering aangenomen dat de persoon deze partner altijd heeft gehad.
+Rule: Gegevens van (ex-)partners kunnen geraadpleegd worden op een specifieke peildatum (in het verleden).
+        - Filteren op peildatum wordt gedaan op basis Datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) en Datum ontbinding huwelijk/geregistreerd partnerschap (07.10).
```

Comment:
"wordt gedaan op basis" is nogal vaag. 

Zoiets?: Bij vragen van partnerhistorie op peildatum wordt de partner geleverd waarbij de datum huwelijkssluiting op of voor de peildatum ligt én de eventuele datum ontbinding na de peildatum.

Reply:
Aangepast